generated/nidcpower/session.py

```diff
@@ -4151,6 +4151,599 @@ def export_attribute_configuration_file(self, file_path):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    @ivi_synchronized
+    def create_advanced_sequence(self, sequence_name, sequence, set_as_active_sequence=True):
+        '''create_advanced_sequence
+
+        Test.
+
+        Note: Test.
+
+        Args:
+            sequence_name (str): Name of sequence.
+
+            sequence (list of int): Sequence.
```

Comment:
This is wrong

Reply:
Please see PR description: 'Documentation and tests still remaining but looking for feedback before doing that work'